packages/driver/src/cypress/mocha.ts

```diff
@@ -326,11 +413,25 @@ function patchTestClone () {
     ret._testConfig = this._testConfig
     ret.id = this.id
     ret.order = this.order
+    ret._currentRetry = this._currentRetry
 
     return ret
   }
 }
 
+function createCalculateTestStatus (Cypress: Cypress.Cypress) {
+  // Adds a method to the test object called 'calculateTestStatus'
+  // which is used inside our mocha patch (./driver/patches/mocha+7.0.1.dev.patch)
+  // in order to calculate test retries. This prototype functions as a light abstraction around
+  // 'calculateTestStatus', which makes the function easier to unit-test
+  Test.prototype.calculateTestStatus = function () {
+    let retriesConfig = Cypress.config('retries')
+
+    // @ts-expect-error
+    return calculateTestStatus.call(this, retriesConfig?.experimentalStrategy, retriesConfig?.experimentalOptions)
```

Comment:
```suggestion
    return calculateTestStatus(this, retriesConfig?.experimentalStrategy, retriesConfig?.experimentalOptions)
```

To go along with changing the test status.

Reply:
addressed in 3ec51b3ce8